js/classes/FlexibleMonthCalendar.js

```diff
@@ -708,10 +708,13 @@ class FlexibleMonthCalendar extends BaseCalendar
 
         function lessThanTwoEntries(index)
```

Comment:
```suggestion
        function lessThanThreeEntries(index)
```
I guess this should be renamed now, shouldn't it?
Other than that, patch looks fine